content/docs/tech-resources/using-proxy-https-tracking-domain.md

```diff
@@ -54,6 +54,9 @@ sudo apt-get install nginx
 
 On a Debian distribution, this command will install nginx with a sample configuration, located at **/etc/nginx/**.  To enable a reverse proxy back to SparkPost for your tracking domain, see the sample configuration file below (sample tracking domain is click.nddurant.com).
 
+Note: we need to set the `proxy_pass` directive with a variable so the ip address of the domain will be looked up
+dynamically and result will be cached for 5 minutes. Otherwise, it would be only looked up at server startup
```

Comment:
Can you please reword this to:
> Note: you must store spgo.io in a variable so that nginx re-resolves the domain when its TTL expires.

I believe that you also need to include the `resolver` directive when using this method [1]. Please check me on that.

[1] https://www.nginx.com/blog/dns-service-discovery-nginx-plus/

Reply:
There's another NGINX config snippet later in this document that we need to update, too.